db/migrate/20211222192244_update_combined_advanced_filters.rb

```diff
@@ -0,0 +1,140 @@
+class UpdateCombinedAdvancedFilters < ActiveRecord::Migration[6.1]
```

Comment:
really nice job with this migration! I've done some pretty extensive testing with the saved filters and have had no issues